fraud_online/streamlit_app.py

```diff
@@ -66,35 +65,33 @@ def plot_pie(values, labels):
 
 st.write(36 * "-")
 print_fancy_header('\n🤖 Connecting to Model Registry on Hopsworks...')
-@st.cache(suppress_st_warning=True)
+@st.cache()
 def get_deployment(project):
     ms = project.get_model_serving()
     deployment = ms.get_deployment("fraudonlinemodeldeployment")
-    deployment.start()
+    deployment.start(await_running=180)
     return deployment
 
 deployment = get_deployment(project)
 
 progress_bar.progress(85)
 
-
 st.write(36 * "-")
 print_fancy_header('\n🧠 Interactive predictions...')
 with st.form(key="Selecting cc_num"):
     option = st.selectbox(
          'Select a credit card to get a fraud analysis.',
-         (X_test.cc_num.sample(5).values)
+         [4915141400146609, 4515188652242507, 4467360740682089, 4705647145380059, 4963241168009873]
```

Comment:
will these number be present any time user runs this tutorial? Can we add some function to fetch cc numbers from the fs?